src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheck.java

```diff
@@ -49,6 +49,7 @@ public ClassFanOutComplexityCheck() {
     public int[] getRequiredTokens() {
         return new int[] {
             TokenTypes.PACKAGE_DEF,
+            TokenTypes.IMPORT,
```

Comment:
We could but in this case we need to change title of issue , as we can not hide from user new functionality, too much people rely on our "log of changes" in release notes.
Let me think about this.